package.json

```diff
@@ -187,6 +187,28 @@
           "type": "boolean",
           "default": true,
           "markdownDescription": "When enabled, if you press the return key from the first line of a multiline string containing a pipe, it will automatically add `.stripMargin`."
+        },
+        "metals.fallbackScalaVersion": {
+          "type": "string",
+          "default": "",
+          "enum": [
+            "2.13.4",
+            "2.13.3",
+            "2.13.2",
+            "2.13.1",
+            "2.13.0",
+            "2.12.13",
+            "2.12.12",
+            "2.12.11",
+            "2.12.10",
+            "2.12.9",
+            "2.12.8",
+            "2.11.12",
+            "3.0.0-M3",
+            "3.0.0-M2",
+            "3.0.0-M1"
```

Comment:
Let's add at least in the main Metals repo `docs/contributors/releasing.md` information that we should update this list.

Reply:
Added to the same PR -  [commit](https://github.com/scalameta/metals/pull/2417/commits/8a9549bd4e0909f26f5e1392f3e64c7a8cfd4621)